react/components/admin/redirects/Form/utils.ts

```diff
@@ -62,7 +62,21 @@ export const getStoreUpdater: StoreUpdaterGetter = operation => (
       writeRedirectsToStore(newData, store)
     }
   } catch (err) {
-    console.log('No cache found for "Redirects".')
+    console.warn('No cache found for "Redirects".')
```

Comment:
I'm not sure this should be a warning; it's just a message that shows up when no cache was found, such as when creating a new entry.